PowerUp/app/src/main/res/layout-land/gamemap.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/map">
+
+    <Button
+        android:id="@id/HouseButton"
```

Comment:
When you define a new id in the `android:id` field you should use `@+id`. Use `@id` elsewhere where you just reference an already-existing id